bin/job_schedulers.py

```diff
@@ -49,6 +49,13 @@ def submit_job(self, logger, build_log_dir, job):
             f"--output={joinpath(build_log_dir, 'slurm-%j.log')}",
         ]
         slurm_keys = {k: v for k, v in tags.items() if k.startswith('slurm_')}
+
+        use_default_reservation = "slurm_reservation" not in slurm_keys
+        # Remove the reservation key if it is False or empty string
+        if 'slurm_reservation' in slurm_keys and not bool(slurm_keys['slurm_reservation']):
```

Comment:
```suggestion
        if slurm_keys.get('slurm_reservation', "false").lower() == "false":
```
This allows both `false` and `False` (and `FALSE`)